### DISCUSSION TITLE
Is it possible to restart a Codespace that has crashed?

### DISCUSSION THREAD

**Discussion by triozer** [community/community#26844] [Codespaces]
<p>Hello :wave:</p>
<p>I was working on a C project, and I ruined the program with an infinite loop… now I can’t access the Codespace anymore. I wish I could get it back since I hadn’t pushed my changes yet…</p>
<p>Is there an existing solution (e.g. rebooting the remote server?) or do I need to recreate a new Codespace and forget the old one?</p>
<p>Here is the VSCode error:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b49d133d2c152a46e3ec2658bed1fe10326a738b.png" data-download-href="/uploads/short-url/pLMtydclXO6pwJpqcMriqSaEYTF.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/12460648/181100261-101ac9e9-dd8c-4863-a21c-61ed8653ec92.png" alt="image" data-base62-sha1="pLMtydclXO6pwJpqcMriqSaEYTF" width="690" height="102" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b49d133d2c152a46e3ec2658bed1fe10326a738b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">904×134 12.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lostintangent** [comment, marked as answer]
<p>Hey! We don’t currently expose a UI gesture to suspend a Codespace, but that is coming soon. In the meantime, the Codespace will auto-suspend itself after 30 minutes of inactivity. And so if you wait a bit and then try to reconnect, that should hopefully unblock you. Could you try that and let us know?</p>

**Comment by triozer** [comment]
<p>Oh that’s great, it works! :sparkles:</p>
<aside class="quote no-group" data-username="lostintangent" data-post="2" data-topic="134446">
<div class="title">
<div class="quote-controls"></div>
 lostintangent:</div>
<blockquote>
<p>We don’t currently expose a UI gesture to suspend a Codespace</p>
</blockquote>
</aside>
<p>Super!</p>
<p>Thank you all for your excellent work :heart:<br>
Hope to hear more about Codespaces in the next days.</p>
